src/histolab/filters/image_filters.py

```diff
@@ -829,7 +1098,22 @@ class GreenPenFilter(ImageFilter):
     -------
     PIL.Image.Image
         Image the green pen marks filtered out.
-    """
+
+
+
+    Example:
+        >>> from PIL import Image
+        >>> from histolab.filters.image_filters_functional import ApplyMaskImage, GreenPenFilter, RgbToGrayscale, OtsuThreshold
```

Comment:
this import won't work cause they're not in the `image_filters_funcitonal` but in `image_filters`